recipes/careamics/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "careamics" %}
+{% set version = "0.0.4.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/careamics-{{ version }}.tar.gz
+  sha256: a1261fa0d552fd0d8fe0a956c64525957fd80fdd0906ef112028f9eeefaf030e
+
+build:
```

Comment:
This looks also like ut could be noarch

Reply:
Hi!

There is a dependency on `pytorch-lightning` and `pytorch`, and as far as I can tell none of them have `Win64` artefacts in the conda-forge channel. So when I had the `noarch` setting in the recipe, the CI failed on Windows. There seem to be an issue with memory in building the torch artefacts on the Windows runner (if I understood correctly the discussions).

I cannot modify the CI files to remove the `win64` CI (which I think the feedstock of both `pytorch` and `pytorch-lightning` that I see on conda-forge did) because it would fail the linter, and it is probably not great practice.

If I leave the CI alone, and simply skip `win64` from the yaml file, then the linter complains that `noarch` cannot have os-restrictions.

Am I missing something? I would love to make it noarch, but the dependencies are necessary in the `import careamics`.

Reply:
Failing CI on Windows with noarch packages due to missing dependencies is quite common and OK. Please make it noarch again.